src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: left;
+  gap: $days-gap;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $day-block-size;
+    height: $day-block-size;
+    border: 1px solid $color-black;
+    background-color: $bg-main-color;
+    transition: all 0.5s linear;
+    color: $color-black;
+
+    &:hover {
+      background-color: $color-pink;
+      cursor: pointer;
+      transform: translateY(-20px);
```

Comment:
Would be better to decrease translateY to -10px because first row is out of the field of view after during :hover effect or add paddings for `.calendar` or `section`

Reply:
@vasylmiskiv 
but on task we need :
`Move the hovered cell up by 20px (use transform)`

Reply:
> or add paddings for `.calendar` or `section`